schema/plugin.json

```diff
@@ -14,6 +14,11 @@
       "title": "Show notification when PYFLYBY starts adding imports to notebook",
       "description": "Inform when pyflyby starts to add imports for the first time. Since this adds imports to the top cell, it may not be obvious to user",
       "default": false
+    },
+    "lockTimeout": {
+      "type": "number",
+      "title": "The timeout (in sec) for when the import locks should expire. If a lock expires then the import will be skipped being added.",
```

Comment:
Why such a long timeout? If pyflyby ever stops responding for a message, doesn't this basically break it forever? With the is busy check, maybe we can use a smaller number like 10s?

I'm also not sure we need to expose this timeout right now. Do you think users will want to tune this or is it an escape hatch?

Reply:
A smaller timeout also works, I've changed it to 10s

I only exposed this as a safety hatch. We can rename this to something like - `experimentalLockTimeout` to make users aware that this may change in the future?

Reply:
We can just break it in a breaking release can keep it as lockTimeout